core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/versionmapper/CamelForWildflyFuse7ToBomMapper.java

```diff
@@ -36,8 +36,8 @@ public CamelForWildflyFuse7ToBomMapper() {
 		fallbackMappings.put(CamelForFuseOnOpenShiftToBomMapper.FUSE_740_CAMEL_VERSION, "5.3.0.fuse-740022-redhat-00002");
 		fallbackMappings.put(CamelForFuseOnOpenShiftToBomMapper.FUSE_750_CAMEL_VERSION, "5.3.0.fuse-750026-redhat-00001");
 		fallbackMappings.put(CamelForFuseOnOpenShiftToBomMapper.FUSE_760_CAMEL_VERSION, "5.4.0.fuse-760021-redhat-00001");
-		fallbackMappings.put(CamelForFuseOnOpenShiftToBomMapper.FUSE_770_CAMEL_VERSION_CAMEL_2_21, "5.5.0.fuse-770010-redhat-00003");
```

Comment:
the version was wrong for Camel 2.21 but I think that it needs to be kept and fixed, not removed

Reply:
spun off another Jira for that: https://issues.redhat.com/browse/FUSETOOLS-3390